### ISSUE TITLE
Add section about the community

### ISSUE THREAD

**Issue by chrislovecnm** [aws-samples/aws-workshop-for-kubernetes#278]
## Why

Kubneretes and kops community is AMAZING.  Support for #kops on slack is amazing.  sig-aws is a great support and fun topics.  We need should have a document talking about how to get involved.  Where to file issues, where to get help, how to get involved!

## Topics

- slack
- kubernetes repos
  - k/k
  - k/kops
  - k/community
- stackoverflow

/cc @parispittman

**Comment by arun-gupta** [comment]
@chrislovecnm Sounds good, send a PR?

**Comment by chrislovecnm** [comment]
Yep ... I cc'ed a community guru as well.  Not sure if you have met @parispittman or Sarah.  If not I will introduce you.

**Comment by arun-gupta** [comment]
A PR from any committer anyone would be great!
